po/nl.po

```diff
@@ -0,0 +1,212 @@
+# Dutch translations for com.github.aggalex.contacts package.
+# Copyright (C) 2019 THE com.github.aggalex.contacts'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.aggalex.contacts package.
+# Automatically generated, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.aggalex.contacts\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-07-22 00:06+0900\n"
+"PO-Revision-Date: 2019-07-22 00:12+0900\n"
+"Last-Translator: Martin Stork <martin.stork95@gmail.com>\n"
+"Language-Team: none\n"
+"Language: nl\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
```

Comment:
Please replace `ASCII` to `UTF-8` here:

```suggestion
"Content-Type: text/plain; charset=UTF-8\n"
```

Otherwise msgfmt seems to output so many errors

Reply:
@ryonakano 最高ですよ！